### ISSUE TITLE
Should be able to draw dot regardless of pen status

### ISSUE THREAD

**Issue by samuel-1184** [datarho/iturtle#18]
Repro:

```python
from turtle import Turtle
from turtle import Screen
from random import randint

screen = Screen()
screen.colormode(255)

turtle = Turtle()

turtle.penup()

for i in range(8):
    r = randint(0, 255)
    g = randint(0, 255)
    b = randint(0, 255)
    turtle.dot(20, (r, g, b))
    turtle.forward(50)
```

But there are no dots with iturtle.

**Comment by samuel-1184** [comment]
Please check #17

**Comment by linbendatarho** [comment]
Fixed in 0.4.1
